doc/theoretical_description_classification.rst

```diff
@@ -6,15 +6,179 @@
+
+.. math:: 
+    s_i(X_i, Y_i) = 1 - \hat{\mu}(X_i)_{Y_i}
+
+Once the conformity scores :math:`{s_1, ..., s_n}` are estimated for all calibration points, we compute the :math:`(n+1)*(1-\alpha)/n` quantile
+:math:`\hat{q}` as follows : 
+
+.. math:: 
+    \hat{q} = Quantile \left(s_1, ..., s_n ; \frac{\lceil(n+1)(1-\alpha)\rceil}{n}\right) \\
+
+
+Finally, we construct a prediction set by including all labels with a score higher than the estimated quantile :
+
+.. math:: 
+    \hat{C}(X_{test}) = \{y : \hat{\mu}(X_{test})_y \geq 1 - \hat{q}\}
+
+
+This simple approach allows us to construct prediction sets coming with a theoretical guarantee on the marginal coverage.
+However, although this method generally results in small prediction sets, it tends to produce empty ones when the model is uncertain,
+for example at the border between two classes.
+
+
+2. Adaptive Prediction Sets (APS)
+---------------------------------
+
+The so-called Adaptive Prediction Set (APS) method overcomes the problem encountered by the LABEL method through the construction of
+prediction sets which are by definition non-empty.
+The conformity scores are computed by summing the ranked scores of each label, from the higher to the lower until reaching the true
+label of the observation :
+
+.. math:: 
+   s_i(X_i, Y_i) = \sum^k_{j=1} \hat{\mu}(X_i)_{\pi_j} \quad \text{where} \quad Y_i = \pi_j 
+
+
+The quantile :math:`\hat{q}` is then computed the same way as the score method.
+For the construction of the prediction sets for a new test point, the same procedure of ranked summing is applied until reaching the quantile,
+as described in the following equation : 
+
+
+.. math:: 
+   \hat{C}(X_{test}) = \{\pi_1, ..., \pi_k\} \quad \text{where} \quad k = \text{inf}\{k : \sum^k_{j=1} \hat{\mu}(X_{test})_{\pi_j} \geq \hat{q}\}
+
+
+By default, the label whose cumulative score is above the quantile is included in the prediction set.
+However, its incorporation can also be chosen randomly based on the difference between its cumulative score and the quantile so the effective
+coverage remains close to the target (marginal) coverage. We refer the reader to [2, 3] for more details about this aspect.
+
+
+3. Top-K
+--------
+
+Introduced by [3], the specificity of the Top-K method is that it will give the same prediction set size for all observations.
+The conformity score is the rank of the true label, with scores ranked from higher to lower.
+The prediction sets are build by taking the :math:`\hat{q}^{th}` higher scores. The procedure is described in the following equations : 
+
+.. math:: 
+   s_i(X_i, Y_i) = j \quad \text{where} \quad Y_i = \pi_j \quad \text{and} \quad \hat{\mu}(X_i)_{\pi_1} > ... > \hat{\mu}(X_i)_{\pi_j} > ... > \hat{\mu}(X_i)_{\pi_n}
+
+
+.. math:: 
+    \hat{q} = \left \lceil Quantile \left(s_1, ..., s_n ; \frac{\lceil(n+1)(1-\alpha)\rceil}{n}\right) \right\rceil
+
+
+.. math:: 
+   \hat{C}(X_{test}) = \{\pi_1, ..., \pi_{\hat{q}}\} 
+
+
```

Comment:
Could be interesting as with the other methods to make a mention of the coverage and possible intuition of this method?

Reply:
You're right, I added a small paragraph at the beginning of the page to introduce the mathematical setting.